bayesian/default_config.py

```diff
@@ -45,6 +45,11 @@
 BAYESIAN_COMPONENT_TAGGED_COUNT = os.getenv('BAYESIAN_COMPONENT_TAGGED_COUNT', 2)
 
 F8_API_BACKBONE_HOST = os.getenv('F8_API_BACKBONE_HOST', 'http://f8a-server-backbone')
+
+SNYK_API_TOKEN_VALIDATION_URL = os.getenv('SNYK_API_TOKEN_VALIDATION_URL',
+                                          'https://snyk.io/api/v1/verify/token')
+ENCRYPTION_KEY_FOR_SNYK_TOKEN = os.getenv('ENCRYPTION_KEY_FOR_SNYK_TOKEN', 'SNYK')
```

Comment:
I think you can add this to the snyk secrets

Reply:
Ok